### PR TITLE
Fix: Editor menu items not working [ED-8168]

### PR DESCRIPTION
**Pull request by yossi-elementor**

### CODE REVIEW

File: assets/dev/scss/editor/panel/_menu.scss [hunk 1]
```diff
@@ -25,7 +25,6 @@
 	&-item {
 		display: flex;
 		align-items: center;
-		padding: 10px;
```
Comment (ManorHazaz) [inline assets/dev/scss/editor/panel/_menu.scss:28]: Will break the budge.

**Review by ghost — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/panel/_menu.scss [hunk 2]
```diff
@@ -39,6 +38,7 @@
 		&-icon {
 			text-align: center;
 			width: 30px;
+		  	padding: 10px 0 10px 10px;
```
Comment (StyleShit) [inline assets/dev/scss/editor/panel/_menu.scss:41]: Is it RTL friendly?

File: assets/dev/scss/editor/panel/_menu.scss [hunk 3]
```diff
@@ -63,7 +63,12 @@
 		}
 
 		a {
+		  	display: flex;
+		  	align-items: center;
 			color: $editor-dark;
+		  	width: 100%;
+		  	height: 100%;
+		  	padding: 10px 10px 10px 0;
```
Comment (StyleShit) [inline assets/dev/scss/editor/panel/_menu.scss:71]: same as above

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: assets/dev/scss/editor/panel/_menu.scss (hunk 2 above)
Comment (yossi-elementor) [reply assets/dev/scss/editor/panel/_menu.scss:41]: It is now. Thanks!

File: assets/dev/scss/editor/panel/_menu.scss (hunk 3 above)
Comment (yossi-elementor) [reply assets/dev/scss/editor/panel/_menu.scss:71]: Same as above ;)

File: assets/dev/scss/editor/panel/_menu.scss [hunk 4]
```diff
@@ -39,7 +38,9 @@
 		&-icon {
 			text-align: center;
 			width: 30px;
-			margin-inline-end: 10px;
+		  	flex-shrink: 0;
+		  	padding: 10px 0;
```
Comment (ManorHazaz) [inline assets/dev/scss/editor/panel/_menu.scss:42]: Redundant.

**Review by ghost — APPROVED** [review]

**Pull request merged**